### PR TITLE
About us workspace

### PR DESCRIPTION
**Pull request by olasores**
My current progress on the about us section, it is responsive not converted to tailwind yet.

### CODE REVIEW

**Comment by 26samaahmed** [comment]
A few things to note so far before you switch to tailwind:
- Can you clear the Awards.svelte and NavBar.svelte files so they're empty as it may conflict since other people are working on those specific features?
- I don't think we need different folders in the routes folder to import the component and use it. You can import in page.svelte and then do ```<About />``` so you can delete the home and about folder in the routes directory.
- The title and text can be a lot thicker. Make sure to import the font like this in the head tag and adjust the font weight.

```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700;800;900&display=swap" rel="stylesheet">
```

- For the containers below the about us title that includes more info, there's a lot of white space on both sides so what we can do is expand the width of the containers and make the font bigger so it looks fuller. Same issue with the 3 containers below 'What can we look forward to?'. We can expand the space between each of the 3 containers as well as stick the text to the bottom. I will leave a few code review changes and you can make more adjustments based on that and the notes above.

 Good Progress 😎

**Comment by 26samaahmed** [comment]
A few things to note so far before you switch to tailwind:
- Can you clear the Awards.svelte and NavBar.svelte files so they're empty as it may conflict since other people are working on those specific features?
- I don't think we need different folders in the routes folder to import the component and use it. You can import in page.svelte and then do ```<About />``` so you can delete the home and about folder in the routes directory.
- The title and text can be a lot thicker. Make sure to import the font like this in the head tag and adjust the font weight.

```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700;800;900&display=swap" rel="stylesheet">
```

- For the containers below the about us title that includes more info, there's a lot of white space on both sides so what we can do is expand the width of the containers and make the font bigger so it looks fuller. Same issue with the 3 containers below 'What can we look forward to?'. We can expand the space between each of the 3 containers as well as stick the text to the bottom. I will leave a few code review changes and you can make more adjustments based on that and the notes above.

Once changes are made, we can review it again and then it can be transferred to tailwind.

 Good Progress 😎

File: src/components/About.svelte [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        </div>
+    </div> -->
+  
+
+    <!-- 3 colummn grid template -->
+    <h1 class="abt">What Can We Look Forward To?</h1>
+     <div class="expectation-wrapper">
+        <div class="column-card">
+            <img src="" alt="">
+            <p><span>challenge and climb with </span>fellow designers</p>
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
            <p><span>Challenge & Climb with </span>fellow designers</p>
```

File: src/components/About.svelte [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        </div>
+        <div class="column-card">
+            <img src="" alt="">
+
+            <p> <span>clash for top prices from our </span>sponsors</p>
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
            <p> <span>Clash for top prices from our </span>sponsors</p>
```

File: src/components/About.svelte [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+        </div>
+        <div class="column-card">
+            <img src="" alt="">
+            <p> <span>get industry feedbacks from</span> mentors</p>
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
            <p> <span>Get industry feedbacks from</span> mentors</p>
```

File: src/components/About.svelte [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+        flex-direction: column; 
+        align-items: center;    
+        justify-content: center; 
+        text-align: center;     
+        padding: 10px;  
+    }
+
+    .abt {
+        color: #6BD1A9;
+        font-size: 4rem;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        font-size: 5rem;
```

File: src/components/About.svelte [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+        font-weight: 900;
+        text-transform: uppercase;
+        -webkit-text-stroke-width: 0.5px;
+        -webkit-text-stroke-color:#1C8B82 ;
+    }
+
+    .abt-fully {
+        color: #959A25;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        color: #959A25;
        font-size: 3rem;
```

File: src/components/About.svelte [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+    }
+
+    .abt-fully-p {
+        color: #294105;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        color: #294105;
        text-align: left;
        font-size: 18px;
```

File: src/components/About.svelte [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+        display: flex;
+        justify-content: center;
+        padding: 20px;
+        width:100%;
+        gap: 10px;
+    }
+
+    .what-container {
+        background-color: #F5ECDE;
+        width: 25%;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        width: 40%;
        height: 450px;
        border-radius: 50px;
        padding: 40px;
        margin: 30px;
```

File: src/components/About.svelte [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+        height: 80%;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
```

File: src/components/About.svelte [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+        border-radius: 40px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
```

File: src/components/About.svelte [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+        padding: 20px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
```

File: src/components/About.svelte [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+        margin: 20px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
```

File: src/components/About.svelte [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+        border: 5px solid #6ED5A6;
+        padding: 10rem;
+        border-radius: 40px;
+        height: 100px;
+        width: 100px;
+    }
+    .expectation-wrapper {
+        display: grid;
+        grid-template-columns: repeat(3,1fr);
+        gap: 10px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        gap: 20px; 
```

File: src/components/About.svelte [hunk 13]
```diff
@@ -0,0 +1,238 @@
...
+        border-radius: 40px;
+        height: 100px;
+        width: 100px;
+    }
+    .expectation-wrapper {
+        display: grid;
+        grid-template-columns: repeat(3,1fr);
+        gap: 10px; 
+        padding: 30px;
+        width: 50%;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        width: 80%;
```

File: src/components/About.svelte [hunk 14]
```diff
@@ -0,0 +1,238 @@
...
+        height: 300px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        height: 400px;
```

File: src/components/About.svelte [hunk 15]
```diff
@@ -0,0 +1,238 @@
...
+        color: #4E7411;
+        -webkit-text-stroke-width: 0.8px;
+        -webkit-text-stroke-color:#6ED5A6;
+    }
+
+    span {
+        color: white;
+        -webkit-text-stroke-width: 0.8px;
+        -webkit-text-stroke-color:#6ED5A6;
+        padding: 5px;
```
Comment (26samaahmed) [inline src/components/About.svelte]:
```suggestion
        padding: 5px;
        font-size: 2rem;
```

**Review by 26samaahmed — CHANGES_REQUESTED** [review]
These are some small detail changes in the CSS

**Comment by tylermlui** [comment]
## I have a slight suggestion with the responsiveness of the images. When it gets to lg"1024px" it seems that the aspect ratio of the images on each column differ: <br>
image
<br>
### You can add  ```lg:max-h-52 xl:max-h-none``` on each image to add a max height of 208px so it will only have a max from lg(1024px) to xl(1280px). 
```
    <div class="p-4 sm:p-6 md:p-8">
        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 sm:gap-8 md:gap-12 mx-auto max-w-7xl">
            <div class="bg-[#B5E8FF] border-2 border-white font-bold rounded-2xl w-full h-[410px] flex flex-col justify-center items-center hover:scale-105  transform hover:bg-[#9DCCE4] hover:border-[#83B0CC] transition-transform duration-300">
                <img class="w-full h-3/4 lg:max-h-52 xl:max-h-none object-contain  p-4 sm:p-5" src="{fully_the_frog_fly}" alt="">
                <p class="m-0 text-[#114674]  text-xl sm:text-2xl px-2.5 text-center"><span>Challenge and Climb with </span>fellow designers</p>
            </div>
            <div class="bg-[#B5E8FF] border-2 font-bold border-white rounded-2xl w-full h-[410px] flex flex-col justify-center items-center  hover:scale-105 transform hover:bg-[#9DCCE4] hover:border-[#83B0CC] transition-transform duration-300">
                <img class="w-full h-3/4 lg:max-h-52 xl:max-h-none object-contain p-4 sm:p-5" src="{fully_once_the_frog}" alt="">
                <p class="m-0 text-[#114674] text-xl sm:text-2xl px-2.5 text-center"><span>Clash for top prices from our </span>sponsors</p>
            </div>
            <div class="bg-[#B5E8FF] border-2 font-bold border-white rounded-2xl w-full h-[410px] flex flex-col justify-center items-center  hover:scale-105 transform hover:bg-[#9DCCE4] hover:border-[#83B0CC] transition-transform duration-300">
                <img class="w-full h-3/4 lg:max-h-52 xl:max-h-none object-contain p-4 sm:p-5" src="{funna}" alt="">
                <p class="m-0 text-[#114674]  text-xl sm:text-2xl px-2.5 text-center"><span>Get industry feedbacks from</span> mentors</p>
            </div>
        </div>
    </div>
```

### It will look like: 



image

## I also have a suggestion for the div that contains the text for "what is fully beyond" and the ACM logo. It seems that when the screen size goes past xl(1280) the div that holds the text gets larger in relation to the screen size:
<br>

image

### You can add ```max-w-screen-2xl``` to the outer div:
```    <div class="flex flex-wrap flex-col  md:flex-row font-bold justify-center p-4 sm:p-5 md:p-8 gap-4 max-w-screen-2xl">
        <div class="bg-[#F5ECDE] w-full md:w-full lg:w-3/5 h-auto rounded-3xl p-5 m-5">
            <h2 class="text-[#959A25] text-2xl md:text-3xl lg:text-4xl  ">What is FullyBeyond?</h2>
            <p class="text-[#294105]  text-base md:text-lg lg:text-xl xl:text-2xl py-4">
                Fully Beyond is the CSUF Designathon 2024 theme, inviting you to leapfrog conventional design thinking and explore the unexplored. 
                Here, we'll delve deep into the vibrant world of inclusivity and diversity as the cornerstones of our design process.
            </p>

            <p class="text-base md:text-lg lg:text-xl xl:text-2xl text-[#294105]">
                Just like a lush rainforest teems with life, design flourishes when it embraces a variety of perspectives. 
                We believe in empathy, equity, and celebrating differences. By going Fully Beyond, we can unlock the power of
                design to shape a future that reflects the richness of our ever-evolving society.
            </p>
        </div>
```
### This will also be the same width as the grid below that holds the images. You can add padding to the text/change font size to make it look a little better as well.
<br>

image

**Comment by 26samaahmed** [comment]
The about us section wasn't exactly centered on mobile so I did some adjustments. Look at video for reference. I think this looks good to merged now! Good job @olasores !!!

https://github.com/user-attachments/assets/e25ad86e-b004-48c7-b97e-efe9afb93a4a

**Pull request merged**